.travis.yml

```diff
@@ -1,44 +1,48 @@
 language: python
-# see https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905
-dist: xenial
```

Comment:
Does 3.7 really work without this?

Reply:
It should, xenial is now the default configuration